src/gwt/org/bbop/apollo/gwt/client/dto/TrackInfo.java

```diff
@@ -84,6 +71,28 @@ public void setPayload(JSONObject payload) {
         this.payload = payload;
     }
 
+    public String getCategory() {
+        return category;
+    }
+
+    public void setCategory(String category) {
+        this.category = category;
+    }
+
+    public String getStandardCategory(){
+        if(category==null || category.trim().length()==0){
+            return "-----------";
```

Comment:
Agreed.  Kind of struggled with this one.  I think I had Uncategorized before.  I think that makes the most sense.